RELEASE_NOTES.md

```diff
@@ -12,9 +12,9 @@
 - Creating configurations just became easier. Using a DSL you can now create configurations in a more declarative and concise way:
 ```Kotlin
 CheckoutConfiguration(
+    shopperLocale = shopperLocale,
     environment = environment,
     clientKey = clientKey,
-    shopperLocale = shopperLocale,
     amount = amount,
     analyticsConfiguration = createAnalyticsConfiguration(),
```

Comment:
I don't think we should have this line at all 😄

Reply:
Removed.

Reply:
Sorry I was referring to the `analyticsConfiguration` line here

Reply:
oh i misunderstood, fixed now